Pendulum_1-added

PetrFD opened the pull request:
Сделал первый маятник (не затухающий), оформил в виде сабсистемы, переменные берутся из workspace

PetrFD committed:
Pendulum_1-added

NoMonsters reviewed (changes requested):
Привет. Да модель работает. Есть несколько замечаний.

1. Integrator, Second-Order - редко используется. Обычно строят через последовательность обычных блоков Integrator. Сдейлай через них
2. В качестве упражнения сконфигурируй модель на использование метода интегрирования с потсоянным шагом (0.01 секунда)
После этого переходи к маятнику с затуханием.

PetrFD committed:
Pendulum-updated

PetrFD commented on Models/training/train_model_PetrFD.slx:
1) Заменил Integrator, Second-Order на два обычных интегратора.
2) Настроил фиксированный шаг интегрирования в Solver равный 0.01

PetrFD committed:
Добавлен затухающий маятник

1)добавлена подсистема с моделью затухающего маятника.
2)Основные параметры моделей записываются в Simulink SDI
3)добавлена строгая типизация входов в подсистему

NoMonsters reviewed (changes requested):
Привет.
Модели работают, но они плохо читаемы.
- На верхнем уровне какие-то 2 сабсистемы, не понтяно что их этого что. Нужно подписать: одна с затуханием другая без
- В модели с затуханием тоже спагети. Модели читаем слева направо, соответственно все повороты в обратную сторону, а тем более ты повернул входы сверху вниз - нечитаемо. Нужно что-то с минимумом пересечений и изломов. И подписать сигналы необходимые для понимания модели.

NoMonsters commented:
Пример читаемой модели 
![изображение](https://github.com/user-attachments/assets/67104f34-ee62-4597-8923-3d0c954d8cc8)

NoMonsters commented:
Ещё нужно дополнить инициализационным скриптом. В модели есть переменные g и l, они никак не инитятся и я как юзер, который просто открыл модель и хочу ей воспользоваться не понимаю что мне делать и откуда взять.
Поправь это тоже, пожалуйста и дальше наверное перейдем уже к микроконтроллерам.

PetrFD committed:
Добавил скрипт и улучшил читаемость моделей

PetrFD committed:
Подписал сигналы

PetrFD committed:
Мигающий светодиод(STM32)
